### ISSUE TITLE
Followups received randomly

### ISSUE THREAD

**Issue by Riesuu** [glpi-project/glpi#8929]
Randomly, some users all different do not receive our follow-ups. All users have an email address set up and functional on GLPI.

In the notifications of follow-ups, the "requester" is well informed.
![image](https://user-images.githubusercontent.com/17006344/113425836-d459c400-93d2-11eb-9bbf-fce35b9c7290.png)

In the queue, the email is not sent at all to the requester
![image](https://user-images.githubusercontent.com/17006344/113425820-cf951000-93d2-11eb-8365-d3594c02f128.png)

The ticket was sent to my colleague, who moved it to the collector's inbox

I don't see anything in the logs, no errors, no calls to the "4477" ticket

bests regards.

**Comment by Riesuu** [comment]
Hello,
I just did another test but the email still doesn't appear in the queue. I added myself and then it worked. Nothing in the logs.

---

Bonjour,
je viens de refaire un test mais l'email n'apparait toujours pas dans la file d'attente. Je me suis ajouté et là ça a pu fonctionner. Rien dans les logs..

**Comment by Riesuu** [comment]
I found the problem.

I have 8 entities:
A
A>A
A>B
A>C

The ticket had been opened on entity "A" and the person is on entity A>B. I had to transfer the ticket to the right entity so that the mail could be sent.  Why is this?

The problem is that I have 500 tickets....
How can I do it?

----


J'ai trouvé le problème.

Je dispose de 8 entités:
A
A>A
A>B
A>C

Le ticket, avait été ouvert sur l'entité "A" et la personne est sur l'entité A>B. J'ai du transférer le ticket sur la bonne entité pour que le mail puisse s'envoyer.  Pourquoi?

Le problème est que j'ai 500 tickets....
Comment faire? merci
